serve/mlc_serve/engine/base.py

```diff
@@ -30,6 +30,7 @@ class MLCServeEngineConfig:
     min_decode_steps: int = 32
     max_decode_steps: int = 48
     init_timeout: int = 120
+    model_type: str = "tvm"  # "tvm", "torch"
```

Comment:
@sunggg This change in config probably requires an update to ollm as well the next time we bump the rev.

Reply:
Or for now we can remove the change in `MLCServeEngineConfig` and hard-code to use the TVM model in `PagedCacheModelModule`. This lets us merge this PR without worrying about breaking ollm or blocking other PRs to be merged. We can add `model_type` config when we introduce PT model support.